script/Arcueid/Day 7/07_00_ARC07_3_1_3.txt

```diff
@@ -637,10 +706,11 @@ That's good to hear.
 [sha:94e70032f2d795edfcf9c9d166e791b8d85b418b]{
 -- Page 619, Offset 18140.
 -- というか、一つの組織が100年かけた計画を返り討ちにするような怪物にほいほいやってこられても困る。
-Or rather, it would have been troublesome if a monster that could reverse a century of planning by a single organization went out on a rampage.
+We'd really be in trouble if the sort of monster that can overturn a plan hundreds of years in the making
+ were to blithely show up now.
 }
 [sha:780e3ac380b75b73ae94e1fce5ffb556e22840c8]{
 -- Page 619, Offset 18141.
 -- それこそ自衛隊の出番ではないだろうか。
-I wonder if that's where the Japanese Self-Defence Force would come in.
+If they did, maybe we'd be better off calling in the Japanese Self-Defence Force?
```

Comment:
If they did, I guess we'd be better of calling in the Japanese Self-Defense Force.